openedx_events/event_bus/__init__.py

```diff
@@ -1 +1,83 @@
 """Classes and utility functions for the event bus."""
+
+from abc import ABC, abstractmethod
+from typing import NoReturn, Optional
+
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+
+class EventBus(ABC):
+    """
+    Parent class for event bus implementations.
+    """
+
+    @abstractmethod
+    def send(
```

Comment:
Just confirming that this replaces the previously documented `send_to_event_bus`, since this would probably result in code like `event_bus.send(...)`?
Related, do we like using the term `send` rather than `produce`, to provide overlap with the signals, since this takes a signal? Or would something like `produce_from_signal` be more clear?

Reply:
No strong opinion! But yeah, it seemed redundant to use the existing name.